src/routes/(inner)/api/use-dismiss/Example.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	import { useFloating, useInteractions, useDismiss } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
+	const click = useDismiss(floating.context);
```

Comment:
Using `const click`, but if we're trying to mimic the Floating UI React docs, this should be `const dismiss`. Then ensure this is passed down to `userInteractions()`